financialaid/api_test.py

```diff
@@ -41,7 +40,9 @@ class FinancialAidBaseTestCase(TestCase):
     @classmethod
     def setUpTestData(cls):
         with mute_signals(post_save):
-            cls.profile = ProfileFactory.create()
+            cls.profile = ProfileFactory.create(
+                country=FuzzyText(length=2)
```

Comment:
I have an issue written for this since it seems like it might be a slightly bigger refactor to ensure country codes are 2 letters long in more than just one model. #1309